personio-personnel-data-api-oa3.yaml

```diff
@@ -2202,7 +2392,11 @@ components:
                     example: 60
                   comment:
                     type: string
-                    example:
+                    example: ""
```

Comment:
We should always feel the examples ... otherwise, you motivate the consumers to leave that field empty making it pretty much obsolete ..